projects/kalidea/kaligraphi/src/lib/molecules/kal-select/kal-select.component.ts

```diff
@@ -318,7 +327,7 @@ export class KalSelectComponent
   /**
    * Event emitted when several options are selected
    * Set the option as active
-   * @param option KalOptionComponent
+   * @param options KalOptionComponent
```

Comment:
if you provide js doc for function args, you should explain their purpose and not type ( which are already set by Typescript )